### PR TITLE
Added docker commands to readme.md

### LINKED ISSUE #51: Add docker commands to README.md
Add instructions to README.md with docker commands to pull and run the application.
Use code markup with 3 ` to make it easy to copy the instructions one by one and run them in a terminal.

```
example instruction that can be copied.
```

### PR DESCRIPTION
**Pull request by aeldin**
This PR updates the README.md file with instructions for running the application with docker

----

| <a href="https://ellipsis.dev" target="_blank">Ellipsis</a> | :rocket: This PR description was created by [Ellipsis](https://www.ellipsis.dev) for commit f33bcddac220d208ad754a3deeb18bed4cd7f3db.  | 
|--------|--------|

### Summary:
This PR updates the Docker instructions and commands in the `README.md` file and corrects a typo in the Docker image name.

**Key points**:
- Updated Docker instructions in `README.md`
- Modified Docker commands in `README.md`
- Corrected Docker image name from `fungover/thuder:latest` to `fungover/thunder:latest`

----
Generated with :heart: by [ellipsis.dev](https://www.ellipsis.dev)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,3 +41,17 @@ This software is licensed under the MIT License, allowing you to use, copy, modi
...
 For more details, refer to the [MIT License](LICENSE) file.
 
+## Running the Application with Docker
+
+Follow these steps to run the application using Docker.
+
+#### Step 1: Pull the Docker Image
+
+\`\`\`
+docker pull thunder:latest
```
Comment (kappsegla) [inline README.md:51]: The address for our image is fungover/thunder:latest

File: README.md [hunk 2]
```diff
@@ -41,3 +41,17 @@ This software is licensed under the MIT License, allowing you to use, copy, modi
...
+\`\`\`
+#### Step 2: Run the Docker Container
+
+\`\`\`
+docker run -p 1883:1883 -d thunder:latest
```
Comment (kappsegla) [inline README.md:56]: Same here. Must be fungover/thuder:latest to work since that is the image name after pulling it.

**Review by kappsegla — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -41,3 +41,17 @@ This software is licensed under the MIT License, allowing you to use, copy, modi
...
 For more details, refer to the [MIT License](LICENSE) file.
 
+## Running the Application with Docker
+
+Follow these steps to run the application using Docker.
+
+#### Step 1: Pull the Docker Image
+
+\`\`\`
+docker pull fungover/thuder:latest
```
Comment (kappsegla) [inline README.md:51]: Missing an n in the thunder. Try the commands in a console window while running docker on the computer to make sure they work.

File: README.md [hunk 4]
```diff
@@ -41,3 +41,17 @@ This software is licensed under the MIT License, allowing you to use, copy, modi
...
+\`\`\`
+#### Step 2: Run the Docker Container
+
+\`\`\`
+docker run -p 1883:1883 -d fungover/thuder:latest
```
Comment (kappsegla) [inline README.md:56]: Here too. Missing an n.

**Review by kappsegla — CHANGES_REQUESTED** [review]

**Review by kappsegla — APPROVED** [review]

**Review by marcusju23 — APPROVED** [review]

**Pull request merged**